docs/content/getting-started/docker.md

```diff
@@ -27,10 +27,10 @@ $ docker-compose up
 To build the Ogmios image from sources, pass the `--build` flag. This is useful if you need a different version than the latest one available on Dockerhub.  
 
 {{% notice tip %}}
-If you're building locally using `docker build`, make sure to leverage existing cache steps from Docker Hub setting the env `DOCKER_BUILDKIT=1` and passing `--cache-from ktorz/ogmios:latest`. For `docker-compose build` or `docker-compose up --build`, **also** set`COMPOSE_DOCKER_CLI_BUILD=1`. A full build of Ogmios without cache may take up to 45 minutes!
+If you're building locally using `docker build --target ogmios`, make sure to leverage existing cache steps from Docker Hub setting the env `DOCKER_BUILDKIT=1` and passing `--cache-from ktorz/ogmios:latest`. For `docker-compose build` or `docker-compose up --build`, **also** set`COMPOSE_DOCKER_CLI_BUILD=1`. A full build of Ogmios without cache may take up to 45 minutes!
```

Comment:
The cache is working for me when not targeting `ogmios`. Is there a reason you're scoping this?

Reply:
I've been trying to keep the first part of the docker doc about "standalone ogmios", while the second part is more about building/running "cardano-node-ogmios". Obviously, this ain't clear enough :sweat_smile: ... The entire page should probably be re-written now since this PR changes quite a lot what we can do with docker.